__tests__/app.test.js

```diff
@@ -70,3 +71,30 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+
+describe("GET /api/articles", () => {
+  test("gives status of 200 and responds with all articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { allArticles } }) => {
+        allArticles.forEach((article) => {
+					expect(Object.keys(article)).toHaveLength(8);
```

Comment:
Rather than using Object.keys here, could we just check the length of allArticles outside the forEach?